augur/filters/include_by_name.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from augur.filters.base_filter import BaseFilter
+
+
+def is_enabled(args):
+    return args.include
+
+
+def add_arguments(parser):
+    parser.add_argument(
+        "--include",
+        type=str,
+        help="file with list of strains that are to be included regardless of priorities or subsampling",
+    )
+
+
+def init(context):
+    if not os.path.isfile(context.args.include):
+        raise  # TODO
+
+    with open(context.args.include, "r") as file:
+        included_names = [
+            # comments may appear inline, so take only the portion of the line before a COMMENT_CHAR
+            line.split(context.COMMENT_CHAR)[0].strip()
```

Comment:
`str.partition` is faster than `str.split` if you only care about a single match:
```
In [7]: %timeit ("Foo" * 100 + "#bar").split("#")[0]
286 ns ± 3.69 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [8]: %timeit ("Foo" * 100 + "#bar").partition("#")[0]
121 ns ± 2.37 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```
(I mean, not a lot faster, but every little bit helps, right?)